### PR TITLE
Less confusing displayed dates for data and metadata, fix #266

### LINKED ISSUE #266: Less confusing displayed dates for data and metadata

### PR DESCRIPTION
**Pull request by davidbgk**
+ retrieve quality field (full dataset) for search results

capture d ecran 2015-11-24 a 09 39 54

### CODE REVIEW

File: udata/core/dataset/api_fields.py [hunk 1]
```diff
@@ -160,6 +160,8 @@
 
 dataset_full_fields = api.inherit('DatasetFull', dataset_fields, {
     'quality': fields.Raw(description='The dataset quality', readonly=True),
+    'last_update': fields.ISODateTime(
```
Comment (noirbizarre) [inline udata/core/dataset/api_fields.py]: May be interesting to always have this information ?

**Comment by noirbizarre** [comment]
r+wc

File: udata/core/dataset/api_fields.py (hunk 1 above)
Comment (davidbgk) [inline udata/core/dataset/api_fields.py]: I hesitated but given that `last_update` is a computation which iterate on each resource, I eventually chose to put it in the `full` version. Waiting for fields-on-demand to fix that once and for all ;-).

Comment (noirbizarre) [inline udata/core/dataset/api_fields.py]: OK for me. If we need it before fields on demand we'll move it.

**Pull request merged**